### PR TITLE
Documentation Update

### LINKED ISSUE #120: Proj Transition:  Documentation
Create link on home page to documentation, review content, link references, etc.
Embed new motion_guide.html and motion.1 into code base.

### PR DESCRIPTION
**Pull request by Mr-DaveDev**
Documentation update to reflect new project and closes #120 This pull also includes the static pictures which are part of the Motion guide.  Including them allows for the use of the guide offline and eliminates privacy link while reviewing guide.  (Debian packager revision with 3.2.12+git20140228-5)

### CODE REVIEW

File: motion_guide.html [hunk 1]
```diff
@@ -215,13 +206,8 @@
 <ul>
 
 <p></p>
-While Motion is included in the apt packages for Ubuntu and Debian, it can also be built from the source code.  
+While Motion is included in the apt packages for Ubuntu and Debian, it can also be built from the See also the <a href="https://github.com/Motion-Project/motion">source code</a>
```
Comment (jogu) [inline motion_guide.html]: Think the "See also the" is superfluous?

File: motion_guide.html [hunk 2]
```diff
@@ -3395,8 +3379,7 @@
 
 <p></p>
 The video4linux driver is written by the same author that first created Motion. You can 
-find the source and a brief description at the 
-<a href="http://www.lavrsen.dk/twiki/bin/view/Motion/VideoFourLinuxLoopbackDevice" class="twikiLink">video4linux loopback device web page</a>. 
+find the source within apt packages.
```
Comment (jogu) [inline motion_guide.html]: I can't actually find the loopback device anywhere other than the old wiki; it used to be packaged in the various linux distress but they all seem to have removed the packages.

I think the two options here are:

1) Remove this section entirely
2) Leave it pointing at the old wiki

I'm probably slightly veering towards 1, unless we find there is a core of users still using it.

**Comment by jogu** [comment]
Everything else looks great to me, this looks like it was a huge amount of work - nice one!

File: motion_guide.html (hunk 2 above)
Comment (Mr-Dave) [inline motion_guide.html]: I've never used this functionality but these are the packages I was seeing in [jessie](https://packages.debian.org/source/jessie/v4l2loopback) and other releases.

Comment (jogu) [inline motion_guide.html]: Ah, I think I was using the wrong search term or something. Fair enough then :)

**Comment by jogu** [comment]
This looks good to me then.

**Pull request merged**